src/Tizen.NUI/src/public/BaseComponents/TextConstants.cs

```diff
@@ -135,6 +140,18 @@ public struct Underline
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public float? Height { get; set; }
+
+        /// <summary>
+        /// The width of the dashes of the dashed underline (if null, the default value is 2.0f). <br />
```

Comment:
One question :
In DALi, the default value is 1 pixel. However, in NUI here, the default value is 2.0f. Is there any reason that they have different values?
-> "If not provided then the default width is used (1 pixel)."
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-toolkit/+/257630/38/dali-toolkit/devel-api/controls/text-controls/text-style-properties-devel.h

Reply:
Because DALi's description is incorrect..
I will fix the description of DALi.

Reply:
Plese refer this.
```
  mUnderlineType(Text::Underline::SOLID),
  mDashedUnderlineWidth(2.0f),
  mDashedUnderlineGap(1.0f),
```
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-toolkit/+/257630/38/dali-toolkit/internal/text/visual-model-impl.cpp

Reply:
Oh I see :)
Thanks for updating DALi description fast.